RomM/api.py

```diff
@@ -478,6 +479,13 @@ def fetch_roms(self) -> None:
                     continue
             if view == View.PLATFORMS and platform_slug != selected_platform_slug:
                 continue
+            # Extract artwork information
+            artwork = {
```

Comment:
Since the 4.4. update, ROMM supports the followingbmedia types:

box2d - Normal cover art (always enabled)
box3d - 3D box art
miximage - Mixed image of multiple media
physical - Disc, cartridge, etc.
screenshot - Screenshot (enabled by default)
title_screen - Title screen
marquee - Custom marquee
logo - Transparent logo
fanart - User uploaded artwork
bezel - Bezel displayed around the EmulatorJS window
manual - Manual in PDF format (enabled by default)
video - Gameplay video (warning: large file size)

Please prefer the miximage as the box art.